rackhd_BIST/bist_utils.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+"""
+RackHD build in self test script
+
+This script will run some test to check RackHD and RackHD running enviroment health status.
+"""
+
+import os
+import sys
+import json
+import time
+import subprocess
+import logging
+
+#LOGGER_NAME = "rackhd_bist_log"
+
+def get_configurations():
+    """
+    Get user BIST configuration from rackhd_bist.conf file
+    User BIST configure will overrride rackhd BIST configure
+    """
+    try:
+        with open("./rackhd_bist.conf") as data_file:
+            rackhd_bist_config = json.load(data_file)
+        with open("./user_bist.conf") as data_file:
+            user_bist_config = json.load(data_file)
+    except IOError:
+        print "Error: can't find rackhd_bist.conf or unable to access it"
```

Comment:
This error may be caused by either `rackhd_bist.conf` or `user_bist.conf`, the error message need be more clear.